KD-1009 - (squashable) fix marcxml fetching in C4::Biblio::Chunker

kohaputti opened the pull request:
the Chunker stopped working since the marcxml column was moved out of
the biblioitems table in Koha community Bug 17196.

paxed commented:
Merged.

johannaraisa commented:
Why there is only exception added and not fixed fetching the marcxml from biblio_metadata?

kohaputti commented:
@johannaraisa I'm not sure what you mean: the function C4::Biblio::GetXmlBiblio I modified the code to use fetches the marcxml from biblio_metadata (check the function definition). I added the exception there so that in the future if C4::Biblio::GetXmlBiblio stops working or there is broken data it will complain and not fail silently.
